lib/matplotlib/axes/_axes.py

```diff
@@ -5496,23 +5496,27 @@ def imshow(self, X, cmap=None, norm=None, *, aspect=None,
             - 'equal': Ensures an aspect ratio of 1. Pixels will be square
               (unless pixel sizes are explicitly made non-square in data
               coordinates using *extent*).
-            - 'auto': The Axes is kept fixed and the aspect is adjusted so
-              that the data fit in the Axes. In general, this will result in
+            - 'auto': The Axes is kept fixed and the aspect is adjusted so that
+              the data fit in the Axes. In general, this will result in
               non-square pixels.
 
         interpolation : str, default: :rc:`image.interpolation`
-            The interpolation method used.
+            The interpolation method used to up- or downsample the image data.
 
             Supported values are 'none', 'antialiased', 'nearest', 'bilinear',
             'bicubic', 'spline16', 'spline36', 'hanning', 'hamming', 'hermite',
             'kaiser', 'quadric', 'catrom', 'gaussian', 'bessel', 'mitchell',
             'sinc', 'lanczos', 'blackman'.
 
-            If *interpolation* is 'none', then no interpolation is performed
-            on the Agg, ps, pdf and svg backends. Other backends will fall back
-            to 'nearest'. Note that most SVG renderers perform interpolation at
-            rendering and that the default interpolation method they implement
-            may differ.
+            The data X is resampled to the pixel size of the image on the
```

Comment:
This is the change.  I don't quite understand why this reflowed.  I had to install black to run boilerplate.py.   I note though that pyplot.py didn't change.  Little confused!